code/datums/quirks/good.dm

```diff
@@ -29,6 +39,48 @@
 	if(mood)
 		mood.mood_modifier += 0.2
 
+/datum/quirk/brusier
+	name = "Bruiser"
+	desc = "You hit a lot harder with your fists"
+	icon = "fist-raised"
+	value = 6
+	gain_text = "<span class='notice'>Your fists feel strong!</span>"
+	lose_text = "<span class='danger'>Your fists feel weaker.</span>"
+	medical_record_text = "Patient displays pugilistic tendancies"
+
+/datum/quirk/brusier/add()
+	if(iscarbon (quirk_holder))
+		var/mob/living/carbon/heavy_hitter = quirk_holder
+		heavy_hitter.dna.species.punchdamagelow *= 1.75
+		heavy_hitter.dna.species.punchdamagehigh *= 1.5
```

Comment:
This doesn't actually do what you might think it does, unless what you want it to do is actually make punches very likely to knock people over. Punch lower for the vast majority of species is 1, so it'll almost always become 1.75, which is then rounded up to 2 when calculating values. Punch upper is almost always 10, which results in the value being 15, which now turns punches into a significantly more swingy affair. It technically raises damage, but it becomes significantly less consistent as the damage is rolled between anywhere from your low values to your high ones.

The real functional bonus this gives is actually making your fists significantly more likely to critical hit, resulting in a knockdown. For humans, stun point for damage is 10, much like their upper damage, so anywhere between 10 and 15 will knock people down. That's a stun proc chance of roughly 35%. That's actually pretty crazy once you factor in stuff like boxing and Gloves of the North Star, and makes having this trait very damn good if you catch yourself without a weapon, since you have a pretty good chance of knocking someone flat and _keeping them down_.

In general, I wouldn't even have quirks mess with species, I'd actually suggest having it change physiology, which would require moving punch values to physiology, like with physiology damage multipliers. Right now, these values are lost on species swap. either that, or use a trait to alter the damage values when it would be rolled in species.